pkg/machines/vmdiskstab.jsx

```diff
@@ -0,0 +1,116 @@
+/*jshint esversion: 6 */
```

Comment:
This adds an entire new file which isn't related to the fix in cockpit-components-listing.jsx nor documented in the commit message. This should probably be split out into a separate commit or even PR, as it doesn't even look related to "fix domstats"?